<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextDialog</class>
 <widget class="QWidget" name="TextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier le texte et choisir le dossier de destination.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_3">
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>431</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Ouvrir &amp;Manager</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>211</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>432</width>
         <height>221</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>Enregistrer &amp;texte modifié</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Enregistrer après choix du nom et prénom</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>  Effacer le fichier présenté et fermer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>441</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Fermer en enregistrant</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
